src/hooks/useTracking.js

```diff
@@ -74,6 +74,7 @@ export const useTracking = (
   }, [openedSections])
 
   useEffect(() => {
+    if (!getConsentStatus()) return
```

Comment:
Note to self: if [PR#217 is merged to master first](https://github.com/wikimedia/wikipedia-kaios/pull/217/commits/f0170f62c9f61121f989aa768a02bc851f0ccbab#diff-d7429549f12cc3c9aa2af3d9d2d16243), revisit where to return early